File: CleverFerretV2/feature/webfiction/src/main/java/com/cleverferret/v2/feature/webfiction/services/webfiction/WebfictionStoryResultV1.java
package com.cleverferret.v2.feature.webfiction.services.webfiction;

import com.cleverferret.v2.feature.webfiction.dto.StoryMetadataDtoV1;

public record WebfictionStoryResultV1(
        String storyId,
        StoryMetadataDtoV1 metadata
) {}

File: CleverFerretV2/feature/webfiction/src/main/java/com/cleverferret/v2/feature/webfiction/services/webfiction/WebfictionStoryResultV1.kt
package com.cleverferret.v2.feature.webfiction.services.webfiction

import com.cleverferret.v2.feature.webfiction.dto.StoryMetadataDtoV1

data class WebfictionStoryResultV1(
    val storyId: String,
    val metadata: StoryMetadataDtoV1
)
